src/pymor/parameters/base.py

```diff
@@ -174,7 +171,7 @@ def parse_value(k, v):
                 return v
             elif isinstance(v, (str, Function)):
                 if isinstance(v, str):
-                    v = ExpressionFunction(v, dim_domain=1, variable='t')
+                    v = ExpressionFunction(v.replace('t', 't[0]'), dim_domain=1, variable='t')
```

Comment:
This is not a good solution. It would replace `tan(t)` with `t[0]an(t[0])`. I understand the desire to write `t` instead of `t[0]`, realizing that properly is not completely trivial: variables in `Expressions` are treated simply as parameters, which necessarily have to be one-dimensional. Only `Expression.to_numpy` it is specified which parameter is actually supposed to be a spatial variable. Maybe we should change that, but I am not completely sure about the consequences atm. Can we remove this feature from this PR and open a separate issue?

Reply:
I remember that something like `sin(x)` worked as an `Expression`, but I guess this was when parameters were allowed to be of any dimension?
I can remove this and open a new issue.